client/extensions/woocommerce/woocommerce-services/state/packages/selectors.js

```diff
@@ -89,7 +89,7 @@ export const getAllSelectedPackages = createSelector(
  * @returns {Object} packages grouped by services
  */
 export const getPackageGroupsForLabelPurchase = createSelector(
-	( state, siteId = getSelectedSiteId( state ) ) => {
+	( state, siteId = getSelectedSiteId( state ), isInternationalOrder ) => {
```

Comment:
I think it would be cleaner (and several other selectors work that way) if you pass `state, siteId, orderId` as arguments, and use those arguments to check if the order is international.

Reply:
👍 

Hehe 😄that's the approach I started with. I switched because none of the packages selectors are aware of the order ID, and I didn't want to add a coupling of the packages state tree and the shippingLabel state tree. But, either way something has to get coupled (either in the selectors, or in the component) and now that I'm re-thinking it, making the selector order-aware as you say is probably better long-term.

Reply:
As an example of a long-term thing that will likely be easier with a selector that's aware of both the packages and shippingLabel: when we add fedex as a shipping service, the list of boxes we'll want to return may depend on a fedex/usps selection, and adding another parameter to the selector will not scale well. Regardless, we'll be able to split the packages and shippingLabel state trees into separate (combined) reducers.